<launch>
  <rosparam command="load" file="$(find location_kdtree)/config/location.yaml" />
  <param name="use_external_carstate" value="true" />

  <node pkg="location_kdtree" type="state_estimator_node" name="state_estimator_node" output="screen">
    <rosparam command="load" file="$(find location_kdtree)/config/state_estimator.yaml" />
  </node>

  <node pkg="location_kdtree" type="location_node" name="location_node" output="screen" />
</launch>
